docs/default-workflow-specs.md

```diff
@@ -0,0 +1,37 @@
+# Default Workflow Spec
+
+![alpha](assets/alpha.svg)
+
+> v2.7 and after
+
+It's possible to set default workflow specs which will be written to all workflows if the spec of interest is not set. This can be configurated through the 
+workflow controller config [map](https://github.com/argoproj/argo/blob/master/workflow/config/config.go#L11) and the field [DefaultWorkflowSpec](https://github.com/argoproj/argo/blob/master/workflow/config/config.go#L69). 
```

Comment:
you should probably make your URL relative

Reply:
Cool, didnt knew you could do that!